addons/web/controllers/main.py

```diff
@@ -701,13 +703,17 @@ def backup(self, master_pwd, name, backup_format = 'zip'):
 
     @http.route('/web/database/restore', type='http', auth="none", methods=['POST'], csrf=False)
     def restore(self, master_pwd, backup_file, name, copy=False):
+        temp_path = tempfile.mkstemp()[1]
```

Comment:
any specific reason to use `mkstemp` rather than a `NamedTempFile`, as done in `exp_restore`?